providers/flagd/pkg/service/in_process/service.go

```diff
@@ -297,6 +285,29 @@ func (i *InProcess) appendMetadata(evalMetadata map[string]interface{}) {
 	}
 }
 
+// makeSyncProvider is a helper to create sync.ISync and return the underlying uri used by it to the caller
+func makeSyncProvider(cfg Configuration, log *logger.Logger) (sync.ISync, string) {
+	if cfg.OfflineFlagSource != "" {
```

Comment:
we could print a INFO log line if both mode have configured parameters 🤔

Reply:
Good suggestion. I added info logs for all service implementation startups. IMO this is a good log to identify the operation mode of the provider